provider/terraform/common~compile.yaml

```diff
@@ -13,6 +13,7 @@
 # These files contains code that needs to be compiled before being delivered to
 # the final module tree structure:
 <% dir = version_name == 'beta' ? 'google-beta' : 'google' -%>
+'website/google.erb': 'provider/terraform/website/google.erb'
```

Comment:
hmm I wonder if this should be called google-beta for google-beta, as well as if all the `docs/google` links should be `docs/google-beta`

Reply:
I think even if it does it could be another additive PR.

Reply:
Oh yeah for sure - totally mean that as a future thing